tools.js

```diff
@@ -59,6 +59,9 @@ module.exports = (function () {
 		timid: {name:"Timid", plus:'spe', minus:'atk'}
 	};
 
+	var init = typeof global.Symbol === 'function' ? Symbol('init') : '_init';
```

Comment:
So why not just name it anything other than `init` or `_init`? I still don't see what symbols are for.